srcpkgs/lokalize/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'lokalize'
+pkgname=lokalize
+version=22.04.3
+revision=1
+build_style=cmake
+configure_args="-DBUILD_TESTING=OFF"
```

Comment:
it fails some tests, did you looked at that?

Reply:
is this one?
```
=> WARNING: No target to "ninja test".
```